modules/nlu/src/backend/tools/vocab-tokenizer.test.ts

```diff
@@ -0,0 +1,59 @@
+import vocabTokenizer from './vocab-tokenizer'
```

Comment:
just call this `getVocabTokenizer` and  straight `vocabTokenizer` instead of `fedVocabTokenizer`